### PR TITLE
Fix libraries for 22.11 pytorch

### PR DESCRIPTION
**Pull request by Tabrizian**

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -221,27 +218,26 @@ if (${TRITON_PYTORCH_DOCKER_BUILD})
     COMMAND docker cp pytorch_backend_ptlib:/usr/local/lib/python3.8/dist-packages/torch/include include/torch
     COMMAND docker cp pytorch_backend_ptlib:/opt/pytorch/pytorch/torch/csrc/jit/codegen include/torch/torch/csrc/jit/.
     COMMAND docker cp pytorch_backend_ptlib:/opt/pytorch/vision/torchvision/csrc include/torchvision/torchvision
-    COMMAND docker cp -L pytorch_backend_ptlib:/usr/local/lib/libopencv_videoio.so libopencv_videoio.so
-    COMMAND docker cp -L pytorch_backend_ptlib:/usr/local/lib/libopencv_highgui.so libopencv_highgui.so
-    COMMAND docker cp -L pytorch_backend_ptlib:/usr/local/lib/libopencv_video.so libopencv_video.so
-    COMMAND docker cp -L pytorch_backend_ptlib:/usr/local/lib/libopencv_imgcodecs.so libopencv_imgcodecs.so
-    COMMAND docker cp -L pytorch_backend_ptlib:/usr/local/lib/libopencv_imgproc.so libopencv_imgproc.so
-    COMMAND docker cp -L pytorch_backend_ptlib:/usr/local/lib/libopencv_core.so libopencv_core.so
+    COMMAND docker cp pytorch_backend_ptlib:/usr/local/lib/libopencv_videoio.so.4.6.0 libopencv_videoio.so
```
Comment (M2Chornyi) [inline CMakeLists.txt:221]: Hi @Tabrizian  
Why we are sticking to the version here instead of following symbolic link?

Comment (Tabrizian) [reply CMakeLists.txt]: Addressed the comment.

File: CMakeLists.txt [hunk 2]
```diff
@@ -161,21 +161,18 @@ if (CMAKE_HOST_SYSTEM_PROCESSOR MATCHES "aarch64")
...
-      "libmkl_core.so"
-      "libmkl_gnu_thread.so"
-      "libmkl_intel_lp64.so"
+      "libiomp5.so"
+      "libmkl_core.so.2"
+      "libmkl_gnu_thread.so.2"
       "libmkl_intel_lp64.so.2"
-      "libmkl_intel_ilp64.so"
-      "libmkl_intel_ilp64.so.2"
-      "libmkl_intel_thread.so"
```
Comment (jbkyang-nvi) [inline CMakeLists.txt:170]: the rest of the intel libraries are not needed?

**Review by mc-nv — APPROVED** [review]

File: CMakeLists.txt (hunk 2 above)
Comment (Tabrizian) [reply CMakeLists.txt:170]: They are needed I just used version .2 instead of the generic one. It seems like new PyTorch version expects version 2.

**Pull request merged**